src/log.h

```diff
@@ -25,32 +25,39 @@
 
 namespace rhost {
     namespace log {
+        enum class log_verbosity {
```

Comment:
It kinda feels like this is the same as `log_level`, just named differently. Do we need these to be separate?